manager/model/machine_model.go

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Delete `the manager/model/machine_model.go` file and use the definitions in `types/model.g. The model directory defines the structure for mysql.